conversations/push-ios/registernotifications/registernotifications.swift

```diff
@@ -0,0 +1,15 @@
+if let updatedPushToken = updatedPushToken {
+  conversationsClient.register(withNotificationToken: updatedPushToken) { (result) in
+    if (!result.isSuccessful()) {
+        // try registration again or verify token
+    }
+  }
+}
+
+if let receivedNotification = receivedNotification {
+  conversationsClient.handleNotification(receivedNotification) { (result) in
+    if (!result.isSuccessful()) {
+        // Handling of notification was not successful, retry?
```

Comment:
Again there is some inconsistent indentation here.

Reply:
Should be addressed.